region/maridia/outer/Fish Tank.json

```diff
@@ -1267,6 +1267,12 @@
         "comesThroughToilet": "any"
       },
       "requires": [
+        {"or": [
+          "Gravity",
+          "HiJump",
+          "canSpringBallJumpMidAir",
```

Comment:
Gravity, HiJump, and Springball will use the more direct strats.  Avoiding the pirate technically has some requirements that you may be avoiding with this shinespark.